WcaOnRails/app/models/country.rb

```diff
@@ -65,16 +69,8 @@ def self.localized_sort_by!(wca_locale, array, &block)
     array.sort_by! { |element| collator.get_sort_key(block.call(element)) }
   end
 
-  ALL_COUNTRIES_WITH_NAME_AND_ID_BY_LOCALE = Hash[I18n.available_locales.map do |locale|
-    countries = localized_sort_by!(locale, Country.all.map do |country|
-      # We want a localized country name, but a constant id across languages
-      # NOTE: it means "search" will behave weirdly as it still searches by English
-      # name (eg: searching for "Tunisie" in French won't match competitions in
-      # "Tunisia" even if the country displayed is actually "Tunisie"...)
-      [country.name_in(locale), country.id]
-      # Now we want to sort countries according to their localized name
-    end) { |localized_name, _id| localized_name }
-
+  ALL_COUNTRIES_BY_LOCALE = Hash[I18n.available_locales.map do |locale|
```

Comment:
- I think that within the `Country` class we can assume the context is countries.
- The purpose should be made clear.

So I'd name this `ALL_SORTED_BY_LOCALE`